.github/workflows/download-image.yml

```diff
@@ -0,0 +1,59 @@
+---
+name: Docker Images
+# on:
+#   schedule:
+#     - cron: "*/15 * * * *"
+#   workflow_dispatch:
+on:
+  push:
+    paths:
+      - '.github/workflows/**'
+jobs:
+  docker-update:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    env:
+      GCR_PAT: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Pull PostGREST Docker Image
```

Comment:
Consider using a matrix strategy here to reduce repetition (eg specify the list of images to scan just once at the top rather than repeating the steps for each below).

Reply:
Was planning to look into this after I get it working.

> I'm mostly OK with merging this if the naming changes are done, but I'm confused about how the scan results will be acted upon; is that something we can resolve before merging? Or does it happen in an immediate followup?

I've addressed the name changes. Will have a follow up PR to make sure that we can use the  scanned images in other jobs and TF, that other jobs are not blocked if scan fails, etc.